Olympus.Stratis/initPlayerLocal.sqf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Author: Theseus Incorporated
+ *
+ * initPlayerLocal.sqf
+ * Initialization of scripts local to player
+ * Executed locally when player joins mission (includes both mission start and JIP)
+ * Executed after initServer and before initPlayerServer and init.
+ *
+ * Arguments: [player:Object, didJIP:Boolean]
+ *
+ */
+
+#include "script_component.hpp"
+
+[] call FUNC(healCommand);
+
+// Types "OLYMPUS" text on the screen, letter by letter, cursor blinking.
+[
+    [
+        ["OLYMPUS", "<t size = '1.5' underline = '1'>%1</t><br/>"],
+        ["Theseus Incorporated"],
+        ["Stratis", "<t size = '1' font='puristaMedium'>%1</t>", 70]
+    ], 1, 0.75, "<t align = 'center' shadow = '1' size = '1.0'>%1</t>"
+] spawn BIS_fnc_typeText;
+
+// Sets default loadout to joining players
+[_this select 0] call FUNC(setBasicLoadout);
```

Comment:
Do a `params ["_player", "_didJIP"];` on top and use it here.
```suggestion
[_player] call FUNC(setBasicLoadout);
```
Refer to template.